### DISCUSSION TITLE
Keep tiles2 and tiles3 support on separate modules

### DISCUSSION THREAD

**Discussion by tdferreira** [ste-gr/struts1-reloaded#31] [Ideas]
Hello,

What do you think about keeping tiles2 and tiles3 on separate modules?
Like this the tiles2 support can still be kept for those that do not want or can't migrate to tiles3.

Maybe just rename the current "tiles2" folder to "tiles3" and have tiles2 load Tiles 2.2.2.

Thanks

**Comment by ste-gr** [comment]
Hi @tdferreira,

sorry for my late reply (holiday). I think it will be a good and easy investment. It would also be nice to solve the current tiles3 problem (#30). I will create a new branch. Maybe you can contribute and/or test it when it's ready.

Greetings
Stefan
